Kinvey/KinveyTests/KinveyTestCase.swift

```diff
@@ -100,9 +156,9 @@ class KinveyTestCase: XCTestCase {
         }
         
         if let username = username {
-            User.signup(username: username, completionHandler: handler)
+            User.signup(username: username, user: user, completionHandler: handler)
```

Comment:
@heyzooi what's the type of `user` here? You should add at least one test where `user` is a custom subclass of `User`.